plugin/pkg/scheduler/factory/factory.go

```diff
@@ -409,11 +600,17 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String,
 		return nil, err
 	}
 
-	// TODO(resouer) use equivalence cache instead of nil here when #36238 get merged
-	algo := core.NewGenericScheduler(f.schedulerCache, nil, predicateFuncs, predicateMetaProducer, priorityConfigs, priorityMetaProducer, extenders)
+	// Init equivalence class cache
+	if getEquivalencePodFunc != nil {
```

Comment:
I would like to avoid enabling it by default now (especially if this will hit 1.7 release).